src/components/follow/Follow.jsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Text, Button, Stack } from "@chakra-ui/react";
+import { Divider } from '@chakra-ui/react';
+import { Avatar } from '@chakra-ui/react';
```

Comment:
You can import all of these in single line.

Reply:
done